src/aleph/vm/orchestrator/status.py

```diff
@@ -120,6 +120,10 @@ async def check_internet(session: ClientSession, vm_id: ItemHash) -> bool:
     try:
         response: dict = await get_json_from_vm(session, vm_id, "/internet")
 
+        if not hasattr(response, "headers"):
```

Comment:
Is there no error message to check instead of looking for a missing key ?

Reply:
If the request times out and don't return a valid response, the key `headers` is missing, so to prevent that I have put that condition to avoid having more issues.